plugins/woocommerce/lib/packages/Detection/MobileDetect.php

```diff
@@ -0,0 +1,1419 @@
+<?php
+/**
+ * Mobile Detect Library
```

Comment:
Can this one be pulled in via Composer (I think this is the [mobiledetect/mobiledetectlib](https://packagist.org/packages/mobiledetect/mobiledetectlib) package)? Or if we're doing a custom fork can we annotate that?

Reply:
Looks like this was manually included due to a bug that had a pending fix: https://github.com/woocommerce/woocommerce/pull/39701/commits/9af74063cbcd6c10554b1935b9c6769a3e58aca6

Since the fix has been merged, reverting to original commit (https://github.com/woocommerce/woocommerce/pull/39701/commits/2baca5a3263dd1fcddc8bb4313903bcb014ba4d9) in https://github.com/woocommerce/woocommerce/pull/39701/commits/9a71cdeb8c4e8fc91b0d8276ef10c4b08b8bcf43